homeassistant/components/zwave_me/sensor.py

```diff
@@ -0,0 +1,114 @@
+"""Representation of a sensorMultilevel."""
+from __future__ import annotations
+
+from zwave_me_ws import ZWaveMeData
+
+from homeassistant.components.sensor import (
+    SensorDeviceClass,
+    SensorEntity,
+    SensorEntityDescription,
+)
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import (
+    ELECTRIC_POTENTIAL_VOLT,
+    ENERGY_KILO_WATT_HOUR,
+    LIGHT_LUX,
+    POWER_WATT,
+    SIGNAL_STRENGTH_DECIBELS,
+    TEMP_CELSIUS,
+)
+from homeassistant.core import HomeAssistant, callback
+from homeassistant.helpers.dispatcher import async_dispatcher_connect
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from . import ZWaveMeController, ZWaveMeEntity
+from .const import DOMAIN
+
+SENSORS_MAP: dict[str, SensorEntityDescription] = {
+    "meterElectric_watt": SensorEntityDescription(
+        key="meterElectric_watt",
+        device_class=SensorDeviceClass.POWER,
+        native_unit_of_measurement=POWER_WATT,
+    ),
+    "meterElectric_kilowatt_hour": SensorEntityDescription(
+        key="meterElectric_kilowatt_hour",
+        device_class=SensorDeviceClass.ENERGY,
```

Comment:
You may want to set state class for the descriptions.

https://developers.home-assistant.io/docs/core/entity/sensor#available-state-classes

https://developers.home-assistant.io/docs/core/entity/sensor#long-term-statistics